smoke-tests/apps/SpringBoot/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/SpringBootControllerSpansEnabledTest.java

```diff
@@ -60,25 +57,7 @@ void testResultCodeWhenRestControllerThrows() throws Exception {
     String operationId = rdEnvelope.getTags().get("ai.operation.id");
     List<Envelope> rddList =
         testing.mockedIngestion.waitForItemsInOperation("RemoteDependencyData", 1, operationId);
-    List<Envelope> edList =
-        testing.mockedIngestion.waitForItems(
-            new Predicate<Envelope>() {
-              @Override
-              public boolean test(Envelope input) {
-                if (!"ExceptionData".equals(input.getData().getBaseType())) {
-                  return false;
-                }
-                if (!operationId.equals(input.getTags().get("ai.operation.id"))) {
-                  return false;
-                }
-                // lastly, filter out ExceptionData captured from tomcat logger
-                ExceptionData data = (ExceptionData) ((Data<?>) input.getData()).getBaseData();
-                return !data.getProperties().containsKey("LoggerName");
-              }
-            },
-            1,
-            10,
-            TimeUnit.SECONDS);
+    List<Envelope> edList = testing.mockedIngestion.waitForItems("ExceptionData", 1);
```

Comment:
let's assert that there was only a single Exception (in both tests), since we want to ensure the duplicate isn't captured